### ISSUE TITLE
[HTML5] multiple builtins reference `_inst` in function calls, which doesn't have them

### ISSUE THREAD

**Issue by thennothinghappened** [YoYoGames/GameMaker-Bugs#4507]
### Description
When using builtins such as (the three I've tested being these), `argument`, `argument_count`, and `working_directory`, HTML5 codegen references them as a property of the `_inst` passed into the function, which doesn't have them, triggering a crash.

### Steps To Reproduce
Here's the example code I used in the submitted project:
```gml
function broken_old_style_argument() {
	
	// "undefined" on HTML5
	show_debug_message(argument_count);
	
	for (var i = 0; i < argument_count; i ++) {
		show_debug_message(argument[i]);
	}
}

function broken_working_directory() {
	
	var dir = working_directory + "/dir";
	show_debug_message(dir);
	
}
```
Calling either of these functions, whether in the script itself, or in an object, will not work. The former does nothing other than print `undefined`, the latter crashes trying to concat `undefined` with `"/dir"`.

Tested in **Beta IDE v2024.200.0.471 Beta Runtime v2024.200.0.487**

[00cb6a82-1c4a-419f-8bf7-5c6c46fbe0b2](https://api.gamemaker.io/api/github/downloads/00cb6a82-1c4a-419f-8bf7-5c6c46fbe0b2)

**Comment by thennothinghappened** [comment]
This also includes the keyword `infinity`.

**Comment by rwkay** [comment]
Fixed in 2024.2

**Comment by mgeddesGM** [comment]
verified as of  IDE v2024.200.0.476  Runtime v2024.200.0.491
